bin/dpd

```diff
@@ -158,7 +158,7 @@ function start(file) {
 
   function startup (err) {
       if (err) {
-        console.log("Failed to start MongoDB");
+        console.log("Failed to start MongoDB (Make sure your -m OR --mongod path is right OR env $PATH points to the mongod executable)");
```

Comment:
If I don't know what's -m, it doesn't help.
I would explain that -m is an argument when you start deployd.
Or even better: just display a link to a dedicated page on our documentation.